envoy/http/header_validator.h

```diff
@@ -77,23 +77,38 @@ class HeaderValidator {
   /**
    * Validate the entire request header map.
    * This method may mutate the header map as well, for example by normalizing URI path.
-   * Returning the Reject value form this method causes HTTP requests to be rejected with 400
-   * status, and gRPC requests with the the INTERNAL (13) error code. Returning the Redirect value
-   * causes HTTP requests to be redirected to the :path presudo header in the request map. gRPC
-   * requests will still be rejected with the INTERNAL (13) error code.
+   * Returning the Reject value form this method causes HTTP request to be rejected with 400
```

Comment:
sorry my correction needs a correction :-(
I was thinking "the HTTP request"
"http requests" was also fine

Reply:
Changed to "the HTTP request".